### ISSUE TITLE
Bijnamen doorzoekbaar maken

### ISSUE THREAD

**Issue by PauluzzS** [lesteenman/pcsa-drupal-theme#119]
Is wel handig om die bijnamenoptie enige relevantie te geven op de site.

**Comment by lesteenman** [comment]
Hier is een zelfde soort tech voor nodig als #12.

**Comment by lesteenman** [comment]
Toch niet helemaal, users zijn niet chill te doorzoeken met een fuzzy filter, denk ik. Ik krijg wel een gecominbeerd filter voor elkaar, om met 1 zoekveld zowel in bijnamen als in de volledige naam te zoeken, maar dan komen er voor een achternaam meerdere resultaten, for some reason.

Een alternatief is om, wanneer je ingelogd bent, een ander form te laten zien, met een onderdeel 'advanced search options', welke standaard dichtgeklapt is. Hier kan o.a. het veld 'bijnamen' in, en eventueel zijn hier andere velden aan toe te voegen. Denk bijvoorbeeld aan woonplaats/adres of werkgever.

**Comment by PauluzzS** [comment]
Dat laatste gaat voorbij aan het doel wat ik ermee had. Ik heb gewoon soms
eerder een bijnaam van een heer in mijn hoofd dan zijn echte naam. Dan is
het handig dat je gewoon de bijnaam kan typon.

Woonplaats zoeken zou overigens wel een feature zijn. Het liefst nog met
een radius optie :P
Maar aangezien ik dat vooral voor KMTs organiseren nuttig zou vinden, is
het wellicht handiger om daar dan een speciale KMT-export-functie van te
maken. Dan kan men de rest gewoon in excel uitzoeken.

Groet,

Paul L. Smits

2017-10-10 10:21 GMT+02:00 Erik <notifications@github.com>:

**Comment by lesteenman** [comment]
Fair enough. Dan zal ik nog een keer kijken of het toch mooier kan. De fuzzy search functie werkt vrij goed voor deze functionaliteit, maar laat zijn resultaten wel altijd zien op een aparte pagina, waardoor je niet in deze lijst blijft.

**Comment by PauluzzS** [comment]
dat is ook stom

Groet,

Paul L. Smits

2017-10-10 12:06 GMT+02:00 Erik <notifications@github.com>:

**Comment by lesteenman** [comment]
Jup, vandaar dat ik die fuzzy search optie al had afgedaan.

**Comment by jpingen** [comment]
> Woonplaats zoeken zou overigens wel een feature zijn. Het liefst nog met
een radius optie :P
Maar aangezien ik dat vooral voor KMTs organiseren nuttig zou vinden, is
het wellicht handiger om daar dan een speciale KMT-export-functie van te
maken. Dan kan men de rest gewoon in excel uitzoeken.

Zoiets kan natuurlijk wel met #6. Overzichtje op maps wie waar woont maakt KMTs plannen wellicht makkelijker.

**Comment by lesteenman** [comment]
Idee van die issue was wel vooral om dat er bij Heeren bij te zetten. Ik had nog niet direct een idee om ergens 1 pagina te maken met een kaart met alle Heeren er op..

**Comment by jpingen** [comment]
Nou, bij dezen het idee dan :p
